package.json

```diff
@@ -4,7 +4,7 @@
   "author": "Tobias Koppers @sokra",
   "description": "Serves a webpack app. Updates the browser on changes.",
   "peerDependencies": {
-    "webpack": ">=2.0.3-beta <3"
+    "webpack": "^2 || 2.0.4-beta || 2.0.5-beta || 2.0.6-beta || 2.0.7-beta || 2.1.0-beta.0"
   },
```

Comment:
Same thing here as my reply to your comment on my pull request.

`^2.1.0-beta.0` also matches `2.1.0` which nobody has tested since it doesn't exist yet.